frontend/main/app/AppController.js

```diff
@@ -26,29 +31,40 @@ export class AppController {
        sellProductsPage: new SellProductsPage(),
      };
 
-     // Set the default page as the marketplace
-     this.#currentView = this.#views.marketplace;
+     // Set the initial view to AuthPage if not logged in, else default page is the marketplace
+     this.#currentView = authService.isLoggedIn() ? this.#views.marketplace : this.#views.auth;
    }
 
-   async render(id=null) {
```

Comment:
The id parameter is still required to navigate to specific product pages, that change should be reversed.